core/sql/regress/qat/dqatdml04.known

```diff
@@ -1,32 +1,12 @@
-2571a2572
-> >>?ignore
-2576,2588d2576
-< 
-< A                     B
-< --------------------  ---------------------
-< 
-<                 2000                2100.00
-<                10000                2100.00
-<                16000                1300.00
-<                16000                2100.00
-<                18000                1600.00
-<                20000                 300.00
-<                20000                2100.00
-< 
-< --- 7 row(s) selected.
-2595,2607c2583
-< 
-< A                     B
-< --------------------  ---------------------
-< 
-<                 2000               2000.000
-<                10000               2000.000
-<                16000               1200.000
-<                16000               2000.000
-<                18000               1500.000
-<                20000                200.000
-<                20000               2000.000
-< 
+2454c2454
+<  100
```

Comment:
Hi Hans,

I did not think I have changed the test. Otherwise, git will pick it up.

I think what really happened is that the old content of the known diff is
not used in recent test. The updated content reflects the new diff.

thanks

2015-07-08 15:29 GMT-05:00 Hans Zeller <notifications@github.com>:

> In core/sql/regress/qat/dqatdml04.known
> <https://github.com/apache/incubator-trafodion/pull/22#discussion_r34194458>
> :
>
> > -< --- 7 row(s) selected.
> > -2595,2607c2583
> > -<
> > -< A                     B
> > -< --------------------  ---------------------
> > -<
> > -<                 2000               2000.000
> > -<                10000               2000.000
> > -<                16000               1200.000
> > -<                16000               2000.000
> > -<                18000               1500.000
> > -<                20000                200.000
> > -<                20000               2000.000
> > -<
> > +2454c2454
> > +<  100
>
> This looks like you changed the test also, removing the ?ignore, but the
> pull request doesn't show a change in the test file. Is that ok?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-trafodion/pull/22/files#r34194458>.
>



-- 
Regards, --Qifan